test/pkg.jl

```diff
@@ -233,10 +233,14 @@ temp_pkg_dir() do project_path
 
     @testset "testing" begin
         Pkg.add(TEST_PKG.name)
+
+        pkgdir = dirname(Base.locate_package(Base.PkgId(TEST_PKG.uuid, TEST_PKG.name)))
+        @test !isnothing(pkgdir)
+        recursive_rm_cov_files(pkgdir) # clean out cov files from previous test runs
+
+        @test !any(endswith(".cov"), readdir(pkgdir)) # should be no cov files to start with
         Pkg.test(TEST_PKG.name; coverage=true)
-        pkgdir = Base.locate_package(Base.PkgId(TEST_PKG.uuid, TEST_PKG.name))
-        # No coverage files being generated?
-        @test_broken TEST_PKG.name * ".cov" in readdir(pkgdir)
```

Comment:
What fixed this?

Reply:
Coverage files now have a PID, so maybe just not looking for the correct files?

Reply:
I see. Makes sense.